frontend/src/queries/nodes/DataNode/dataNodeLogic.ts

```diff
@@ -203,6 +203,8 @@ export const dataNodeLogic = kea<dataNodeLogicType>([
                             if (firstTimestamp) {
                                 const nextQuery: EventsQuery = { ...query, after: firstTimestamp }
                                 return nextQuery
+                            } else {
+                                return query
```

Comment:
This fixes a bug: before, if there were no entries in the initial response, the "automatically load more events" button would not be shown.